lib/sdoc/rdoc_monkey_patches.rb

```diff
@@ -60,3 +60,15 @@ def record_location(*)
     super unless @ignoring_constants
   end
 end)
+
+RDoc::Markup::PreProcess.prepend(Module.new do
+  def find_include_file(name)
+    to_search = [File.dirname(File.dirname(@input_file_name))].concat @include_path
```

Comment:
The following would be nicer but won't work on Ruby 2.7:
```suggestion
    to_search = [File.dirname(@input_file_name, 2)].concat @include_path
```